README.md

```diff
@@ -303,36 +303,36 @@ If there currently isn't an active session, then calling this method does nothin
 Common Features
 ===============
 

```

Comment:
This repository also has a readme.txt file with duplicate content to this file that you may also want to update. That txt file is what's displayed in an IDE after installing the NuGet package.

Reply:
Have edited the .txt file to reflect what's within the README